### PR TITLE
Provide workaround for running multiple Wash shells

### PR DESCRIPTION
**Pull request by MikaelSmith**
Temporary fix for #302.

### CODE REVIEW

File: cmd/rootMain.go [hunk 1]
```diff
@@ -124,12 +117,13 @@ func rootMain(cmd *cobra.Command, args []string) exitCode {
...
-	mountpath, err := ioutil.TempDir(cachedir, "mnt")
+	// Create a temporary run space for aliases and server files.
+	rundir, err := ioutil.TempDir(cachedir, "run")
 	if err != nil {
-		cmdutil.ErrPrintf("Unable to create temporary mountpoint in %v: %v\n", cachedir, err)
+		cmdutil.ErrPrintf("Error creating temporary run location in %v: %v\n", cachedir, err)
 		return exitCode{1}
 	}
-	defer os.RemoveAll(mountpath)
+	defer os.RemoveAll(rundir)
```
Comment (MikaelSmith) [inline cmd/rootMain.go]: This seems to be interacting poorly with the mountpoint.

Comment (ekinanp) [reply cmd/rootMain.go]: What do you mean?

Comment (MikaelSmith) [reply cmd/rootMain.go]: I'm getting lots of "OSXFUSE Volume" icons on my desktop, which I think means they're not being unmounted correctly.

**Comment by MikaelSmith** [comment]
That was my thinking. It was interfering with demos I'm writing, so I wanted a quick fix.

**Pull request merged**